a/afex_hytrix-go/afex_hystrix-go_ubi_8.5.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash -e
+# ----------------------------------------------------------------------------
+#
+# Package       : afex/hystrix-go
+# Version       : v0.0.0-20180502004556-fa1af6a1f4f5
+# Source repo   : https://github.com/afex/hystrix-go
+# Tested on     : UBI: 8.5
+# Language      : GO
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Valen Mascarenhas / Vedang Wartikar <Vedang.Wartikar@ibm.com>
+#
+# Disclaimer: This script has been tested in non-root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+
+PACKAGE_NAME=afex/hystrix-go
+PACKAGE_VERSION=${1:-v0.0.0-20180502004556-fa1af6a1f4f5}
+PACKAGE_URL=https://github.com/afex/hystrix-go
+
+export GOPATH=/go
+export PATH=$GOPATH/bin:$PATH
+
+#Install the required dependencies
+yum -y update && yum install -y git vim wget curl gcc gcc-c++ golang.ppc64le
+
+mkdir -p go/src && cd go/src
+
+#Clone the repository
+git clone https://github.com/afex/hystrix-go.git
+
+
+cd hystrix-go/hystrix
+
+
+
+git checkout fa1af6a1f4f5
```

Comment:
you can use `${1:-fa1af6a1f4f5}` as the value of `PACKAGE_VERSION` and use this variable during `git checkout`. It would be easy to run the script from command line with some different version as the first parameter.